prowler/providers/aws/services/iam/iam_role_cross_service_confused_deputy_prevention/iam_role_cross_service_confused_deputy_prevention.metadata.json

```diff
@@ -0,0 +1,34 @@
+{
+  "Provider": "aws",
+  "CheckID": "iam_role_cross_service_confused_deputy_prevention",
+  "CheckTitle": "Ensure IAM Service Roles prevents against a cross-service confused deputy attack",
+  "CheckType": [],
+  "ServiceName": "iam",
+  "SubServiceName": "",
+  "ResourceIdTemplate": "arn:partition:service:region:account-id:resource-id",
+  "Severity": "high",
+  "ResourceType": "AwsIamPolicy",
+  "Description": "Ensure IAM Service Roles prevents against a cross-service confused deputy attack",
+  "Risk": "Allow attackers to gain unauthorized access to resources",
+  "RelatedUrl": "",
+  "Remediation": {
+    "Code": {
+      "CLI": "",
+      "NativeIaC": "",
+      "Other": "",
+      "Terraform": ""
+    },
+    "Recommendation": {
+      "Text": "Use the aws:SourceArn and aws:SourceAccount global condition context keys in resource-based policies to limit the permissions that a service has to a specific resource",
+      "Url": "https://docs.aws.amazon.com/IAM/latest/UserGuide/confused-deputy.html#cross-service-confused-deputy-prevention"
+    }
+  },
+  "Categories": [],
```

Comment:
This should be part of some categories already...

Reply:
Done